cmake/FindLibUV.cmake

```diff
@@ -13,7 +13,7 @@ endif()
 find_path(LIBUV_INCLUDE_DIR uv.h
   HINTS ${PC_LIBUV_INCLUDEDIR} ${PC_LIBUV_INCLUDE_DIRS})
 
-list(APPEND LIBUV_NAMES uv)
+list(APPEND LIBUV_NAMES uv libuv)
```

Comment:
```suggestion
list(APPEND LIBUV_NAMES uv uv_a)
```
Maybe?

Reply:
I like mine better ;) See

https://github.com/neovim/neovim/runs/7655386228?check_suite_focus=true#step:10:277

Reply:
Yes, I think CMake adds a `lib` prefix automatically, which is why we start by looking for `uv` instead of `libuv`.

Reply:
Right; CMake automatically prepends `lib`. Let's give it a try!